docs/src/site/twiki/FalconDocumentation.twiki

```diff
@@ -91,6 +91,47 @@ Startup property file in both falcon & prism server need to be configured with t
 
 ---++++ Prism Setup
 <img src="PrismSetup.png" height="400" width="600" />
+
+---+++ Safe Mode
+Safemode is useful when the admin wants to prevent Falcon users from scheduling entities in the workflow engine. This can happen when
+   * Hadoop clusters are being upgraded.
+   * Falcon cluster entity is being updated.
```

Comment:
Better to use plurality to be consistent with the previous sentence, i.e. "Falcon cluster entities are being updated".